mmcv/cnn/utils/weight_init.py

```diff
@@ -78,6 +80,7 @@ def bias_init_with_prob(prior_prob):
 class BaseInit(object):
 
     def __init__(self, *, bias=0, bias_prob=None, layer=None):
+        self.overmodule = False
```

Comment:
The name of overmodule is a little bit weird. May not get the meaning at the first glance.